config/environments/development.rb

```diff
@@ -59,4 +59,6 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  config.hosts << "zech.local"
```

Comment:
What's this about?

Reply:
zech.local is my local mac....i just added that to make it easier for me to look  at the app on my iphone on my local network.....should i take it out??

Reply:
Ah hmm yea, since that is specific to your local setup, I’d probably leave that out. If you want to add that as an instruction in the README for others who want to check this out on their local machines, I’d be open to that so that others can accomplish similar without hard coding fixed host names in the repo itself. Or maybe configuring this to use an environment variable instead of hard coding your specific host name in here. Thoughts?

Reply:
I'll just take it out.  i put it in there just to make my life easier and then i made some changes and didn't take it back out....i should have taken it out!